src/plots/cartesian/axis_defaults.js

```diff
@@ -189,3 +210,21 @@ function rangebreaksDefaults(itemIn, itemOut, containerOut) {
         }
     }
 }
+
+// these numbers are one more than what bounds would be mapped to
+var dayStrToNum = {
+    sun: 1,
+    mon: 2,
+    tue: 3,
+    wed: 4,
+    thu: 5,
+    fri: 6,
+    sat: 7
+};
+
+function indexOfDay(v) {
+    if(typeof v !== 'string') return;
+    return dayStrToNum[
+        String(v).substr(0, 3).toLowerCase()
```

Comment:
```suggestion
        v.substr(0, 3).toLowerCase()
```

Reply:
OMG. Fixed in https://github.com/plotly/plotly.js/pull/4661/commits/c48f551fb2c93fe856f340a55c470f6a8c54ea1b.